health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryHome.js

```
@@ -71,23 +78,35 @@
             style={{ width: "35rem", height: "5rem" }}
             textStyles={{ fontSize: "1.5rem" }}
           />)
         })}
         </div>
       </Card>
-      <InfoCard
+      {!disButton && authorized && <InfoCard
         label="Info"
         variant="default"
         style={{maxWidth:"200rem", marginTop:"1rem"}}
         additionalElements={[<span style={{ color: "#505A5F", fontWeight:600 }}>{t(`CURRENT_HIERARCHY_TYPE_IS`)} {": "} {t(data?.hierarchyName)}</span>,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)